test/unit/warn-removed-experimental-config.test.ts

```diff
@@ -104,6 +104,27 @@ describe('warnOptionHasBeenMovedOutOfExperimental', () => {
     expect(config.experimental.foo).toBe('bar')
     expect(config.deep.prop.baz).toBe('bar')
   })
+
+  it('should show the new key name in the warning', () => {
+    const config = {
+      experimental: {
+        bundlePagesExternals: true,
+      },
+    } as any
+
+    warnOptionHasBeenMovedOutOfExperimental(
+      config,
+      'bundlePagesExternals',
+      'bundlePagesRouterDependencies',
+      'next.config.js',
+      false
+    )
+
+    expect(spy).toHaveBeenCalledWith(
+      expect.stringContaining('⚠'),
+      '`bundlePagesExternals` has been moved out of `experimental` and into `bundlePagesRouterDependencies`. Please update your next.config.js file accordingly.'
```

Comment:
What do you think about 
```suggestion
      '`experimental.bundlePagesExternals` has been moved to `bundlePagesRouterDependencies`. Please update your next.config.js file accordingly.'
```

This makes it a bit clearer what "moving out of experimental" means.